featuretools/synthesis/dfs.py

```diff
@@ -242,6 +244,9 @@ def dfs(
     if not isinstance(entityset, EntitySet):
         entityset = EntitySet("dfs", dataframes, relationships)
 
+    if isinstance(cutoff_time, str):
+        cutoff_time = dateutil.parser.parse(cutoff_time)
```

Comment:
What if this raises a ParserError or OverflowError?

Reply:
Thanks. I added handling for parseutil's ParserError and a plain OverflowError (looks like parseutil doesn't have a OverflowError defined, as far as I can tell). In the event of a ParserError is it best to just raise a ValueError?

Reply:
Not sure. @thehomebrewnerd any clue on what we usually do here?

Reply:
I used `dateutil.parser.parse` only because it was in the issue description; I can change it to `pandas.to_datetime` right now. If `pandas.to_datetime` fails, it returns `pd.NaT`. In that case, what would be the most sensible behavior for the function?